nfc/libs/NfcCoreLib/lib/Nci/phNciNfc_Init.c

```diff
@@ -848,136 +768,4 @@ phNciNfc_ResetNtfCb(void*     pContext,
     }
     PH_LOG_NCI_FUNC_EXIT();
     return wStatus;
-}
-
-static NFCSTATUS phNciNfc_RegAllNtfs(void*     pContext)
```

Comment:
Perhaps call `phNciNfc_RegAllNtfs` at the start of `phNciNfc_InitReset`? Then you wouldn't have this inconsistency. Also it would keep all the initialize code together.